multicore_runtime/legacy_e2e_support.py

```diff
@@ -86,18 +86,23 @@ def legacy_app_for_script(script):
     The application as extracted from the script, or None.
   """
   with MockedWsgiHandler.python_25_app_lock:
-    modname, _ = script.rsplit('.', 1)
+    try:
+      modname, _ = script.rsplit('.', 1)
+    except ValueError:
+      modname = script
     app_holder = []
     revert_func = lambda: None
     try:
       revert_func = stub_wsgi_utils(app_holder)
       runpy.run_module(modname, run_name='__main__')
+    except ImportError as e:
+      logging.exception('Error loading %s: %s', script, e)
```

Comment:
not a big thing. Just want to make sure that you are aware that logging.exception will print out the stack trace without giving e here. So, you can choose not to pass in e.